cms/djangoapps/contentstore/xblock_storage_handlers/view_handlers.py

```diff
@@ -387,12 +398,11 @@ def _save_xblock(  # lint-amnesty, pylint: disable=too-many-statements
     publish=None,
     fields=None,
     summary_configuration_enabled=None,
-):
+):  # lint-amnesty, pylint: disable=too-many-statements
```

Comment:
```suggestion
):  # pylint: disable=too-many-statements
```
nit: we shouldn't be adding lint-amnesty to new pylints